src/Appwrite/Platform/Tasks/Restore.php

```diff
@@ -0,0 +1,291 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Exception;
+use Utopia\App;
+use Utopia\DSN\DSN;
+use Utopia\Platform\Action;
+use Utopia\CLI\Console;
+use Utopia\Storage\Device;
+use Utopia\Storage\Device\DOSpaces;
+use Utopia\Storage\Device\Local;
+use Utopia\Validator\Text;
+use Utopia\Validator\WhiteList;
+
+class Restore extends Action
+{
+    protected ?DSN $dsn = null;
+    protected string $database;
+    public const PROCESSORS = 4;
+
+    public static function getName(): string
+    {
+        return 'restore';
+    }
+
+    public function __construct()
+    {
+        $this->checkEnvVariables();
+
+        $this
+            ->desc('Restore a DB')
+            ->param('id', '', new Text(20), 'Folder Identifier')
+            ->param('cloud', null, new WhiteList(['true', 'false'], true), 'Take file from cloud?')
+            ->param('database', null, new Text(10), 'example: db_fra1_01..')
+            ->param('datadir', null, new Text(100), 'mysql datadir path')
```

Comment:
Let's rename `datadir` to be more descriptive

Reply:
Maybe it is not straight forward, but this is the name of the variable MySQL stores the data (/var/lib/mysql).
xtrabackup is using the same name:
https://docs.percona.com/percona-xtrabackup/2.4/xtrabackup_bin/xbk_option_reference.html#-datadirdirectory